### DISCUSSION TITLE
Bug: no editor-toolbar in Markdown editor fullscreen mode

### DISCUSSION THREAD

**Discussion by ReddewYoung** [OrchardCMS/OrchardCore#7160] [Q&A]
Bug: no editor-toolbar in Markdown editor fullscreen mode.

![](https://www.reddewyoung.com/media/images/no-toolbar-bug.png)

[Screenshot](https://www.reddewyoung.com/media/images/no-toolbar-bug.png)

**Comment by hishamco** [comment]
Did you refresh your browser? perhaps some styles and scripts not loaded properly

**Reply by ReddewYoung** [reply]
> Did you refresh your browser? perhaps some styles and scripts not loaded properly

1. I cleared all the browser data;
2. I made a new fresh installation;
3. I got the same result, so I'm sure that's a bug.

**Reply by hishamco** [reply]
If you are working on the nightly builds I will have a look to verify from the bug
